packages/traceability-schemas/schemas/CTPATCertificate.json

```diff
@@ -0,0 +1,32 @@
+{
+  "$id": "https://w3id.org/traceability/schemas/CTPATCertificate.json",
+  "$schema": "https://json-schema.org/draft-07/schema#",
+  "$comment": "{\"term\": \"CTPATCertificate\", \"@id\": \"https://w3id.org/traceability#CTPATCertificate\"}",
+  "title": "CTPAT Certificate",
+  "description": "Customs Trade Partnership Against Terrorism (CTPAT) is but one layer in U.S. Customs and Border Protection’s (CBP) multi-layered cargo enforcement strategy. Through this program, CBP works with the trade community to strengthen international supply chains and improve United States border security. CTPAT is a voluntary public-private sector partnership program which recognizes that CBP can provide the highest level of cargo security only through close cooperation with the principle stakeholders of the international supply chain such as importers, carriers, consolidators, licensed customs brokers, and manufacturers. The Security and Accountability for Every Port Act of 2006 provided a statutory framework for the CTPAT program and imposed strict program oversight requirements.",
```

Comment:
We might want to add a clear disclaimer that these are examples, and hypothetical, and not endorsed by DHS or US CBP.... ideally we can say this once at the top of the spec.

Reply:
Good point, @OR13. I haven't done the "intro" part yet which includes the example (it's only on the PR). I wanted to have #225 merged before changing it further. I'll make sure it includes the wording you suggest.